packages/core/system/src/provider.tsx

```diff
@@ -1,14 +1,18 @@
 import {SSRProvider} from "@react-aria/ssr";
+import {I18nProvider, I18nProviderProps} from "@react-aria/i18n";
 import {OverlayProvider} from "@react-aria/overlays";
 
 export interface NextUIProviderProps {
   children: React.ReactNode;
+  locale: I18nProviderProps["locale"];
```

Comment:
Hey @jguddas Add this as an optional parameter

`locale?: I18nProviderProps["locale"]`

Reply:
```sugesstion
  locale?: I18nProviderProps["locale"];
```